test/test_derivs.jl

```diff
@@ -8,38 +8,48 @@ using Transform
 import Synthetic
 import WCS
 
-import GSL.deriv_central
+import ForwardDiff
+
 
 println("Running derivative tests.")
 
+function do_deriv_test(f::Function, x::Float64, claimed_dx)
+    # TODO: really should call ForwardDiff.gradient just once, with a whole
+    # vector of x values, rather than once per parameter to test
+    fwd_deriv = ForwardDiff.gradient(f)([x,])[1]
+
+    info("got $fwd_deriv; expected $claimed_dx")
+
+    @test_approx_eq_eps significand(fwd_deriv) significand(claimed_dx) 1e-4
```

Comment:
Of course it does, but if we have that kind of floating point calculation in our ELBO calculation then our tests /should/ fail.  (For example, I used autodiff to find one such calculation in the KL divergence, exposing a bug when we were only bounding the mean and variance of the gamma distribution.)

I thought you were doing exact autodiff here, not finite difference differentiation.  If you are, then the results should match to very high precision.  Looking more carefully I'm not confident I understand what you're doing, though, though.  Why are you not just calculating the full gradient with ForwardDiff.gradient and comparing to the sensitive float values?